src/Function.h

```diff
@@ -126,6 +151,9 @@ class Function
 	//GCC C Extensions
 	bool func_attr_inline;
 	bool func_attr_aligned;
+	std::vector<FunctionAttribute*> attributes;
```

Comment:
I don't like it as it lives in every instance of `Function`. But you want to make it static so that there will be just a single instance of the vector (and you then initialize it just once as well).

Reply:
Sure.